doc/tutorial/Client.lhs

```diff
@@ -27,8 +27,8 @@ Also, we need examples for some domain specific data types:
 
 ``` haskell
 data Position = Position
-  { x :: Int
-  , y :: Int
+  { xCoord :: Int
```

Comment:
Fixed now. I also notice there's an educational point in calling the field name `xCoord` and the `Capture` argument `x`, to show that they don't need to be equal.